rasa/nlu/featurizers/dense_featurizer/spacy_featurizer.py

```diff
@@ -60,11 +60,9 @@ def train(
     def get_doc(self, message: Message, attribute: Text) -> Any:
         return message.get(SPACY_DOCS[attribute])
 
-    def process(self, message: Message, attribute: Text = TEXT, **kwargs: Any) -> None:
-        if attribute not in DENSE_FEATURIZABLE_ATTRIBUTES:
-            return
-
-        self._set_spacy_features(message, attribute)
+    def process(self, message: Message, **kwargs: Any) -> None:
```

Comment:
@tabergma I have general question. Every tokenizer computes features and adds this features to the message. While the computation of the features and the setting might be individual for every tokenizer, the implementation of `process` should always be the same (probably also for `train` right?). Would it make sense from your perspective to introduce the 2 methods `get_features` and `set_features` in the `Featurizer` parent class?

Reply:
(wouldn't do it as part of this PR if it even makes sense)

Reply:
I think you are mixing tokenizers and featurizers in your comment. We have this for the tokenizer. I don't think it makes sense for the featurizers as some of them, for example, need to generate a vocabulary first in train before setting features. Also not all featurizers are handling the same kind of text messages, e.g. text vs intent vs response vs action name. But maybe `set_features` makes sense. So I think we might be able to extract some useful helper methods to the super class but I would not implement `process` or `train` over there.